source/extensions/tracers/lightstep/lightstep_tracer_impl.cc

```diff
@@ -141,6 +141,8 @@ LightStepDriver::TlsLightStepTracer::TlsLightStepTracer(
   enableTimer();
 }
 
+LightStepDriver::TlsLightStepTracer::~TlsLightStepTracer() { flush_timer_->disableTimer(); }
```

Comment:
Note that this happens automatically, though this is useful for testing. Up to you whether you want to keep this PR or not. I'm fine either way. cc @rnburn

Reply:
Ah, I didn't notice this logic. 

Sorry for distraction.